app/routes/events/EventCreateRoute.js

```diff
@@ -36,6 +49,51 @@ const mapStateToProps = (state, props) => {
       pools: [],
       unregistrationDeadline: time({ hours: 12 })
     },
+    event: {},
+    pools: [],
+    actionGrant: []
+  };
+
+  /* If there is no eventId in the params then we use the
+   * normal initalValues for a new event
+   */
+  if (!eventId) return initialCreateValues;
```

Comment:
You will need the
```
event: {
     addFee: valueSelector(state, 'addFee'),
etc....
``` here.

otherwise it won't work :/

Reply:
I can't just return an empty object?

Reply:
It looks to working fine in the browser